Makefile

```diff
@@ -83,7 +83,7 @@ lint:
 
 .PHONY: docker-component # Not intended to be used directly
 docker-component: check-component exe
-	docker build -t grafana/$(COMPONENT) --platform=linux/$(GOARCH) --build-arg=TARGETARCH=$(GOARCH) -f ./cmd/$(COMPONENT)/Dockerfile .
+	docker build -t grafana/$(COMPONENT) --build-arg=TARGETARCH=$(GOARCH) -f ./cmd/$(COMPONENT)/Dockerfile .
```

Comment:
i noticed this as well.  i didn't realize that `--platform` was experimental when i merged this.  should we change this in a different PR though?

wondering why you felt the need to change it back.

Reply:
I think we should change this, yeah. I couldn't build on local (I need to run Docker Desktop with experimental features enabled to run this) and thought this would not be the case for most people.

Reply:
Yeah.  I want a separate clean PR to remove this.  It was contributed by @morlay and I really like the functionality, but I have a hard time leaving an experimental Docker feature the default in our Makefile.

Maybe we could have it only set the platform var under certain circumstances?  I wonder if there's a way to detect if Docker supports it or something.

Reply:
may consider set `DOCKER_CLI_EXPERIMENTAL=enabled` before `docker build`